tests/Proto.Cluster.Tests/ClusterTests.cs

```diff
@@ -57,6 +57,14 @@ public async Task HandlesSlowResponsesCorrectly()
         response.Message.Should().Be(msg);
     }
 
+    [Fact]
+    public async Task ThrowsWhenTimeoutExceeded()
+        => await Members.First()
+            .Invoking(m => m.RequestAsync<Pong>(CreateIdentity("slow-test"), EchoActor.Kind,
+                    new SlowPing {Message = "hi", DelayMs = 5000}, new CancellationTokenSource(2000).Token
```

Comment:
The timeout here might be even lower, to provide for a faster test?

Reply:
Moved to separate class and experimented with timeout length. It seems that for netcoreapp3.1 target (where we use TaskClock) 4 seconds is a min for this no to fail in CI.